### PR TITLE
Clean up shell.nix

### PR DESCRIPTION
**Pull request by jacobweinstock**
## Description

clean up shell.nix file of unused/needed deps for CI.

## Why is this needed

Fixes: #

## Checklist:

I have:

- [ ] updated the documentation and/or roadmap (if required)
- [ ] added unit or e2e tests
- [ ] provided instructions on how to upgrade

### CODE REVIEW

File: shell.nix [hunk 1]
```diff
@@ -9,29 +9,11 @@ in { pkgs ? import (_pkgs.fetchFromGitHub {
 
 with pkgs;
 
-let
```
Comment (chrisdoherty4) [inline shell.nix:12]: How does this work with the CI now?

Comment (jacobweinstock) [reply shell.nix:12]: Dont know if I understand the questions, sorry. Currently, the `shell.nix` file is only needed for the `ci-checks.sh` script.

Comment (chrisdoherty4) [reply shell.nix:12]: That answers the question 👍🏻 ; I'd forgotten.

**Review by chrisdoherty4 — APPROVED** [review]

**Comment by chrisdoherty4** [comment]
@mergifyio queue

**Pull request merged**